patches/mas/chromium/no_private_api.patch

```diff
@@ -284,6 +356,62 @@ index dfba0bded9e3..876f96999f53 100644
  }
  
  }  // namespace sandbox
+diff --git a/sandbox/mac/seatbelt_extension.cc b/sandbox/mac/seatbelt_extension.cc
+index 9073364..2356add 100644
+--- a/sandbox/mac/seatbelt_extension.cc
++++ b/sandbox/mac/seatbelt_extension.cc
+@@ -8,6 +8,7 @@
+ #include "base/memory/ptr_util.h"
+ #include "sandbox/mac/seatbelt_extension_token.h"
+ 
++#ifndef MAS_BUILD
+ // libsandbox private API.
+ extern "C" {
+ extern const char* APP_SANDBOX_READ;
+@@ -18,6 +19,7 @@ char* sandbox_extension_issue_file(const char* type,
+                                    const char* path,
+                                    uint32_t flags);
+ }
++#endif
+ 
+ namespace sandbox {
+ 
+@@ -46,7 +48,11 @@ std::unique_ptr<SeatbeltExtension> SeatbeltExtension::FromToken(
+ 
+ bool SeatbeltExtension::Consume() {
+   DCHECK(!token_.empty());
++#ifndef MAS_BUILD
+   handle_ = sandbox_extension_consume(token_.c_str());
++#else
++  handle_ = -1;
```

Comment:
Should this be `handle_ = 0` because of the DCHECK in the dtor https://cs.chromium.org/chromium/src/sandbox/mac/seatbelt_extension.cc?sq=package:chromium&g=0&l=25 ?

Reply:
The MAS build is not supposed to invoke any sandbox code, so it do not make a difference here.

But triggering the DCHECK is probably better since it would warn we have run into code path that should not be invoked.